ci/workflows/pull-request.yml

```diff
@@ -58,6 +58,10 @@ steps:
       - monorepo-diff#v1.2.0:
           diff: "git diff --name-only origin/main"
           watch:
+            - path: "ci/.env"
```

Comment:
Shall we merge the path?

https://github.com/adikari/monorepo-diff-buildkite-plugin#path
> A list of paths can be provided to trigger the desired pipeline. Changes in any of the paths will initiate the pipeline provided in trigger.

Reply:
Let's do it when there're 3 paths 😄